packages/eslint-config-cup-recommended/package.json

```diff
@@ -12,10 +12,10 @@
     "eslint-plugin-prettier": "^2.6.0"
   },
   "peerDependencies": {
-    "eslint": "5.x"
+    "eslint": "^5.0.0 || ^6.0.0"
```

Comment:
Does eslint 5.x support `ecmaVersion: 2018`?

Reply:
Looks like it has been the equivalent of `experimentalObjectRestSpread` since 5 - https://eslint.org/docs/user-guide/migrating-to-6.0.0#-the-depreacted-experimentalobjectrestspread-option-has-been-removed